PORTAL/jobs/__main__.py

```diff
@@ -400,92 +403,96 @@ def cmd_run_next(jobs: Jobs) -> None:
                 # XXX Check the pidfile?
             else:
                 logger.warning('another job is already running, adding %s back to the queue', reqid)
-                jobs.queue.unpop(reqid)
+                jobs.queue(workerid).unpop(reqid)
     except KeyboardInterrupt:
         cmd_cancel(jobs, reqid, _status=Result.STATUS.PENDING)
         raise  # re-raise
 
 
 def cmd_queue_info(jobs: Jobs, *, withlog: bool = True) -> None:
```

Comment:
I was going to suggest optionally passing which queue to show.  Normally it wouldn't be a big deal, but "withlog" may cause significant output so the option to limit to a single queue might help.

Reply:
Done